src/index.html

```diff
@@ -21,7 +21,425 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo__image"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
+              class="nav__item is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__item"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__logo"></div>
+        <div class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars__container">
+              <div class="stars stars--4">
```

Comment:
The nested `stars` block inside another `stars` block can lead to confusion and is not a good BEM practice. Consider renaming the inner block to something more descriptive and unique.